### DISCUSSION TITLE
Pricing Catalog Tables

### DISCUSSION THREAD

**Discussion by nfourtythree** [craftcms/commerce#2344] [Ideas] [answered]
- [ ] Allowed the orderby of  products/variants by salePrice (#1604)

**Comment by TomDeSmet** [comment]
Will this allow us to query on `salePrice` also?

Eg.: I have a price slider to filter products. So I do this in my elementApi:
`$criteria['hasVariant']['price'] = ['and', '>= ' . $priceMin, '<= ' . $priceMax];` 
but I'd like to use the `salePrice` as those will not be filtered correctly now.

**Comment by lukeholder** [comment, marked as answer]
This is now out in in Commerce 5!
